assets/src/story-embed-block/block/index.js

```diff
@@ -18,12 +18,11 @@
  * WordPress dependencies
  */
 import { __ } from '@wordpress/i18n';
-// TODO: Replace with web stories icon.
-import { page as icon } from '@wordpress/icons';
 
 /**
  * Internal dependencies
  */
+import { ReactComponent as icon } from '../../../images/stories.svg';
```

Comment:
After updating the icon svg file, this didnt update. 

<img width="65" alt="Screenshot 2020-06-26 at 15 32 15" src="https://user-images.githubusercontent.com/237508/85868504-46f19f80-b7c2-11ea-994e-c32d36f0aeb9.png">

I am not sure why. Any thoughts @swissspidy

Reply:
Hmm maybe because of Babel cache?

Try `rm -rf ./node_modules/.cache/babel-loader/*` and then `npm run build`